Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -2045,6 +2046,11 @@ else if (!dasSnapshotFile.canRead()) {
       catch (MissingResourceException e) {
         // NOP
       }
+      String sdkVersion = DartSdk.getDartSdk(myProject).getVersion();
```

Comment:
IDE shows a warning in this line. I've reused `sdk` parameter available here instead.